doc/classes/EditorPlugin.xml

```diff
@@ -754,5 +754,14 @@
 		<constant name="DOCK_SLOT_MAX" value="8" enum="DockSlot">
 			Represents the size of the [enum DockSlot] enum.
 		</constant>
+		<constant name="AFTER_GUI_INPUT_PASS" value="0" enum="AfterGUIInput">
+			Forwards the [InputEvent] to other EditorPlugins.
+		</constant>
+		<constant name="AFTER_GUI_INPUT_STOP" value="1" enum="AfterGUIInput">
+			Prevents the [InputEvent] from reaching other Editor classes.
+		</constant>
+		<constant name="AFTER_GUI_INPUT_DESELECT" value="2" enum="AfterGUIInput">
+			Forwards the [InputEvent] to other EditorPlugins with a notification that deselecting is needed.
```

Comment:
Furthering @tinmanjuggernaut's comment, it's not immediately clear how this notification happens. ("Notification" in Godot context would typically refer to actually receiving `NOTIFICATION_*` in `_notification` or maybe a signal.

Reading the code, it seems this doesn't actually do anything specific in EditorPlugin, and is handled only in Node3DEditorPlugin to **bypass** the normal selection logic, is that right?

Reply:
If you keep the normal selection logic in a specific edit mode (PASS), you will deselect the Node you are editing when you select an empty space. However with STOP, the selection logic that use intersect_ray() will not work.
I remember this was necessary to bypass the selection process for the Nodes in the scene while still allowing the intersecting ray on viewport. It might be better to name it something else.

Reply:
How about naming it `AFTER_GUI_INPUT_EXCLUSIVE`?